### PR TITLE
reduce indent to display as regular text

### PR DESCRIPTION
**Pull request by mosesmarin**
reduce indent to display as regular text

### CODE REVIEW

File: tensorflow/python/ops/init_ops_v2.py [hunk 1]
```diff
@@ -636,7 +636,7 @@ def lecun_normal(seed=None):
       seed: A Python integer. Used to seed the random generator.
```
Comment (tamaranorman) [inline tensorflow/python/ops/init_ops_v2.py]: Can you change these as well to make it consistent indentation within the docstring please

File: tensorflow/python/ops/init_ops_v2.py [hunk 2]
```diff
@@ -662,7 +662,7 @@ def lecun_uniform(seed=None):
...
```
Comment (tamaranorman) [inline tensorflow/python/ops/init_ops_v2.py]: And here and the ones below

**Review by tamaranorman — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/init_ops_v2.py (hunk 1 above)
Comment (mosesmarin) [reply tensorflow/python/ops/init_ops_v2.py]: thanks for the feedback @tamaranorman , i have changed the indentation on seed lines indicated and the ones below.

File: tensorflow/python/ops/init_ops_v2.py (hunk 2 above)
Comment (mosesmarin) [reply tensorflow/python/ops/init_ops_v2.py]: changed

**Comment by mosesmarin** [comment]
Added another commit to change indentation in additional lines

**Review by tamaranorman — APPROVED** [review]

**Pull request merged**